### PR TITLE
Closes #363 & Closes #364: Adds private browsing theme

### LINKED ISSUE #364: URL Box View background not shown in Browser Fragment
When we added the ToolbarUIView and ToolbarComponent, we moved the urlBoxView (and similar) to ToolbarUIView, but BrowserFragment doesn't currently use the ToolbarComponent so the url box view background isn't there anymore.

## Steps to reproduce
Look at Browser Fragment 

### Expected behavior
URL background appears

### Actual behavior
No URL Background

### Device information

* Android device: ?
* Fenix version: ?

### PR DESCRIPTION
**Pull request by sblatz**
Let's merge this in favor of https://github.com/mozilla-mobile/fenix/pull/387

### CODE REVIEW

File: app/src/main/java/org/mozilla/fenix/HomeActivity.kt [hunk 1]
```diff
@@ -56,6 +67,28 @@ open class HomeActivity : AppCompatActivity() {
...
+    private fun applyStatusBarTheme() {
+        window.statusBarColor = ContextCompat
+            .getColor(this, DefaultThemeManager
+                .resolveAttribute(android.R.attr.statusBarColor, this))
+
+        window.navigationBarColor = ContextCompat
+            .getColor(this, DefaultThemeManager
+                .resolveAttribute(android.R.attr.navigationBarColor, this))
+
+        when (themeManager.currentTheme) {
```
Comment (sblatz) [inline app/src/main/java/org/mozilla/fenix/HomeActivity.kt]: This is really the only section I have doubts on (the rest has been reviewed/pair programmed). Flipping bits seems unclean, but the window does *not* get recreated when we make a new activity. Is there an easier way to just redraw this window? Perhaps @colintheshots knows 🤔

**Comment by sblatz** [comment]
Also, we should squash & merge this 😄

File: app/src/main/java/org/mozilla/fenix/HomeActivity.kt (hunk 1 above)
Comment (boek) [reply app/src/main/java/org/mozilla/fenix/HomeActivity.kt]: @sblatz I think you can go back to the default value which is just 0 for both of those properties versus flipping the bits

Comment (sblatz) [reply app/src/main/java/org/mozilla/fenix/HomeActivity.kt]: That was my first thought as well. Unfortunately doing that means they get *unset* when using a theme that sets them (as the window is not redrawn after)

**Pull request merged**